src/Resources/Locales/de_DE.axaml

```diff
@@ -479,14 +480,14 @@
   <x:String x:Key="Text.Repository.Refresh" xml:space="preserve">Aktualisiern</x:String>
   <x:String x:Key="Text.Repository.Remotes" xml:space="preserve">REMOTES</x:String>
   <x:String x:Key="Text.Repository.Remotes.Add" xml:space="preserve">REMOTE HINZUFÜGEN</x:String>
-  <x:String x:Key="Text.Repository.Resolve" xml:space="preserve">AUFLÖSEN</x:String>
+  <x:String x:Key="Text.Repository.Resolve" xml:space="preserve">LÖSEN</x:String>
```

Comment:
"Auflösen" sounds a bit more like getting rid of it.

Reply:
What about "beheben"? Or even something like "Konflikte (auf)lösen/beheben" (one version of course 😉)?

Reply:
I am not sure where this text is used so can't check if it'll fit but "Konflikte beheben" is a great translation IMO.